python_ta/typecheck/base.py

```diff
@@ -129,42 +133,83 @@ def lookup_method(name, caller_type, *args):
     return TYPE_SIGNATURES[caller_origin][name]
 
 
+def make_set(node):
```

Comment:
These should all be methods in `TypeConstraints`. Also `make_set` should not take in a node, but an item. It should instantiate a `TNode` given the item.